app/models/user.rb

```diff
@@ -17,8 +23,14 @@ def update_posts_counter
     update(posts_counter: posts.count)
   end
 
-  # this method is called when a user is created to set a default photo
-  def set_default_photo
+  # this method is called when a user is created to set default values (photo and posts_counter)
+  def set_default_values
     self.photo ||= "https://randomuser.me/api/portraits/men/#{rand(1..100)}.jpg"
+    self.posts_counter = 0
+  end
+
+  # define roles for users
+  def admin?
+    role == 'admin'
   end
```

Comment:
- Great work creating `user.admin?` method inside the user model 👏🏿 👏🏿

Reply:
Thank you @peterrobert